documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -562,58 +565,50 @@ Next in our boot sequence is the BTX Server. Let us quickly remember how we got
 * [.filename]#boot0# relocates itself to `0x600`, the address it was linked to execute, and jumps over there. It then reads the first sector of the FreeBSD slice (which consists of [.filename]#boot1#) into address `0x7c00` and jumps over there.
 * [.filename]#boot1# loads the first 16 sectors of the FreeBSD slice into address `0x8c00`. This 16 sectors, or 8192 bytes, is the whole file [.filename]#boot#. The file is a concatenation of [.filename]#boot1# and [.filename]#boot2#. [.filename]#boot2#, in turn, contains the BTX server and the [.filename]#boot2# client. Finally, a jump is made to address `0x9010`, the entry point of the BTX server.
 
-Before studying the BTX Server in detail, let us further review how the single, all-in-one [.filename]#boot# file is created. The way [.filename]#boot# is built is defined in its [.filename]#Makefile# ([.filename]#/usr/src/sys/boot/i386/boot2/Makefile#). Let us look at the rule that creates the [.filename]#boot# file:
+Before studying the BTX Server in detail, let us further review how the single, all-in-one [.filename]#boot# file is created. The way [.filename]#boot# is built is defined in its [.filename]#Makefile# ([.filename]#/usr/src/stand/i386/boot2/Makefile#). Let us look at the rule that creates the [.filename]#boot# file:
```

Comment:
There's a full path used in this this.

Reply:
I have left it as it is in the original document full path used in this one.It Isn't hard for me to fix it if I have to?

Reply:
So should I change that or leave it that way?